app/src/main/java/com/breadwallet/tools/util/BRConstants.java

```diff
@@ -141,6 +141,7 @@ private BRConstants() {
     public static final String _20240101_US = "user_signup";
     public static final String _20241006_DRR = "did_request_rating";
     public static final String _20241006_UCR = "user_completed_rating";
+    public static final String _HOME_OPEN = "home_open";
```

Comment:
this is event to trigger display in-app messaging, will share the video on slack

Reply:
https://firebase.google.com/docs/in-app-messaging/modify-message-behavior?platform=android#trigger_in-app_messages_programmatically_2